### DISCUSSION TITLE
Discussion topics for May 23, 2022 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#5529] [Show and tell]
1. @carlosfelix2 - Reminder about KDoc for backend code
2. @carlosfelix2 - Reminder about code coverage for backend code.

**Comment by jeremy-page** [comment]
On call rotation & duties

**Comment by jeremy-page** [comment]
Unit test madness! (2 Unit tests are failing in GHA but work locally)
